_faq/installation_and_deployment/forseti VM instance security updates.md

```diff
@@ -0,0 +1,14 @@
+---
+title: How do I keep Forseti VM instances (client and server) up to date with security patches? 
```

Comment:
spelling: `up-to-date` is the proper spelling

Reply:
updated